core/store/models/p2pkey/p2p_key.go

```diff
@@ -97,6 +105,16 @@ func CreateKey() (Key, error) {
 	}, nil
 }
 
+func MustCreateKey() Key {
+	p2pPrivkey, _, err := cryptop2p.GenerateEd25519Key(rand.Reader)
```

Comment:
Careful! This is expensive.

Can we find a way to avoid this in tests, perhaps with a fixture?

Reply:
yep for sure